Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM python:3.9
+
+WORKDIR /tmp
+
+# Add, and run as, non-root user.
+RUN mkdir /src
+RUN adduser --disabled-password --gecos "" --shell /bin/bash mitodl
+
+# Install Python packages
+## Set some poetry config
+ENV  \
+  POETRY_VERSION=1.5.1 \
+  POETRY_VIRTUALENVS_CREATE=false \
```

Comment:
```suggestion
```
Looks like this project is using virtual envs with poetry (which imo is good, and simplifies installing packages).